pkg/controller/mustgather/mustgather_controller.go

```diff
@@ -216,9 +217,11 @@ func (r *ReconcileMustGather) Reconcile(request reconcile.Request) (reconcile.Re
 
 	if err != nil {
 		if errors.IsNotFound(err) {
+			// Increment prometheus metrics for must gather total
+			localmetrics.MetricMustGatherTotal.Inc()
```

Comment:
Do we want to wait to do this until after the error check?

Reply:
Good point. Moved to after job creation.